cli/src/version.ts

```diff
@@ -1,7 +1,18 @@
-import { getRevision, getVersion, version as core } from '@ailly/core';
+import { fileURLToPath } from "node:url";
+import { join, normalize } from "node:path";
+import { getRevision, getVersion } from "@ailly/core/dist/version.js";
+import { version as core } from "@ailly/core/dist/index.js";
 
 export function version() {
-  const cli = getVersion(import.meta.url);
-  const rev = getRevision(import.meta.url)
-  console.log(`cli@${cli} core@${core} ${rev}`)
-}
\ No newline at end of file
+  let dirname: string | undefined;
+  try {
+    dirname = __dirname;
+  } catch (e) {}
+  try {
+    dirname = normalize(join(fileURLToPath(import.meta.url), ".."));
```

Comment:
Lol, I hate this as much as I know it's necessary.

Reply:
By the end of this sequence we get entirely to cjs & `try { __dirname }`!